Makefile

```diff
@@ -26,70 +26,113 @@ MAKEFLAGS += -j$(shell nproc)
 #MAKEFLAGS += -Oline
 
 .PHONY: all
-all: check test dist dist-test doc licenseheaders
+all: format check test dist dist-test doc | conda-env
```

Comment:
- Prepares a new `PHONY` target `format` that currently only includes `licenseheaders`.
- Makes `conda-env` an order only prerequisite for all others in this target list.